tests/lava/lib/optimization/solvers/generic/test_solver_cpu_backend.py

```diff
@@ -94,34 +95,33 @@ def test_subprocesses_connections(self):
         )
 
     def test_qubo_cost_defines_weights(self):
+        from lava.lib.optimization.solvers.generic.solution_finder.models\

```

Comment:
Is there a reason to import this here, instead at the top of the file?

Reply:
resolved in upcoming commit